weaviate/weaviate_client.go

```diff
@@ -50,13 +51,13 @@ func NewConfig(host string, scheme string, authConfig auth.Config, headers map[s
 	return &Config{Host: host, Scheme: scheme, Headers: headers, ConnectionClient: client}, nil
 }
 
-func AddAuthClient(cfg Config, authConfig auth.Config, startup_period int) (Config, error) {
+func AddAuthClient(cfg Config, authConfig auth.Config, startupTimeout int) (Config, error) {
```

Comment:
> startupTimeout int 

This should be `time.Duration`, no need to implicitly covert inside this function. It is good UX and idiomatic in Go to have a user just pass in an explicit `time.Duration` whenever setting timeouts, or dealing in elapsed time